libp2p/RLPxHandshake.h

```diff
@@ -58,8 +58,6 @@ class RLPXHandshake: public std::enable_shared_from_this<RLPXHandshake>
 	/// Setup outbound connection.
 	RLPXHandshake(Host* _host, std::shared_ptr<RLPXSocket> const& _socket, NodeID _remote): m_host(_host), m_remote(_remote), m_originated(true), m_socket(_socket), m_idleTimer(m_socket->ref().get_io_service()) { crypto::Nonce::get().ref().copyTo(m_nonce.ref()); }
 
-	~RLPXHandshake() {}
```

Comment:
The class has a virtual protected function, shouldn't it have virtual destructor?